projects/mercury/src/dashboard/DashboardPage.js

```diff
@@ -0,0 +1,76 @@
+import React, {useContext} from "react";
+
+import withStyles from "@mui/styles/withStyles";
+import {Grid, Link, Paper, Typography} from "@mui/material";
+import BreadCrumbs from "../common/components/BreadCrumbs";
+import styles from "./DashboardPage.styles";
+import MetadataViewContext from "../metadata/views/MetadataViewContext";
+import ExternalMetadataSourceContext from "../metadata/external-sources/ExternalMetadataSourceContext";
+import UserContext from "../users/UserContext";
+import DomainInfo from "./DomainInfo";
+import {APPLICATION_NAME, METADATA_VIEW_MENU_LABEL} from "../constants";
+
+const DashboardPage = (props) => {
+    const {currentUser, classes} = props;
+    const {views} = useContext(MetadataViewContext);
+    const {externalMetadataSources} = useContext(ExternalMetadataSourceContext);
+    const canViewMetadata = currentUser && currentUser.canViewPublicMetadata && views && views.length > 0;
+
+    return (
+        <Grid container justifyContent="center" spacing="5">
+            <BreadCrumbs />
+            <Paper className={classes.mainPage}>
+                <Grid container justifyContent="center" spacing="5">
```

Comment:
Will you move it left as discussed?

Reply:
In slack Elisa mentioned she liked centered view. Let's leave it for now. Next week we will have a more in depth design discussion with Elisa and Elena.